datasets/flwr_datasets/resplitter/__init__.py

```diff
@@ -12,13 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
-"""Resplitter package."""
+"""Preprocessor package."""
 
 
 from .merge_resplitter import MergeResplitter
-from .resplitter import Resplitter
 
 __all__ = [
     "MergeResplitter",
```

Comment:
Should we introduce a `MergePreprocessor` class that basically does the same as `MergeResplitter`. Then, keep `MergeResplitter` as a dummy sub-class from the new `MergePreprocessor` that does nothing new but throws a deprecated warning encouraging users to use `MergePreprocessor` instead? wdyt?

Reply:
I thought this could be left as it is. Resplitting is the main action this class (also DivideResplitter) does, but conceptually, it's a Preprocessor (but it does not have that in its name). Wdyt?